src/app/tasks/task.service.ts

```diff
@@ -0,0 +1,52 @@
+import { EventEmitter } from "@angular/core";
+import { Task } from "./task.model";
+import taskData from '../../assets/tasks.json';
+import { TaskList } from "./task-list.model";
+
+export class TaskService {
+
+    taskCreated = new EventEmitter<Task>();
```

Comment:
You can define event emitters in the components themselves with angular @Output then use that to call the service method to do the actions you need to

Reply:
So you would create the outputs in task item component then in the task list component you would listen to that event emitter and call the service method from task list component